libexec/rtld-elf/rtld_c18n.c

```diff
@@ -1333,11 +1373,21 @@ tramp_reflect(void *entry)
 /*
  * APIs
  */
+/*
+ * XXX: Explicit function pointer used so that RTLD can wrap it in trampoline.
+ */
+extern void (*_rtld_sighandler)(int, siginfo_t *, void *);
+extern void (*_rtld_dispatch_signal)(int, siginfo_t *, void *);
+
+void _rtld_sighandler_unsafe(int, siginfo_t *, void *);
+void _rtld_dispatch_signal_unsafe(int, siginfo_t *, void *);
+
 #define	C18N_FUNC_SIG_COUNT	72
 
 void
-c18n_init(void)
+c18n_init(Obj_Entry *obj_rtld)
 {
+	extern Obj_Entry obj_rtld;
```

Comment:
Don't you no longer need the extern if you are passing it in now?  (Presumably this won't compile?)